test/debugger/helper-debugger-repl.js

```diff
@@ -34,11 +34,11 @@ let quit;
 
 function startDebugger(scriptToDebug) {
   scriptToDebug = process.env.NODE_DEBUGGER_TEST_SCRIPT ||
-                  common.fixturesDir + '/' + scriptToDebug;
+                  `${common.fixturesDir}/${scriptToDebug}`;
```

Comment:
I don't think a `path.join` is important here, node modules always use `/` (so it's not an OS compatibility concern). The main advantage of `path.join` is that it abstracts joining complex paths, for concatenating two strings `/` should be fine.

That said, if this view is contested - we can just leave it as is in the PR.